packages/react-native-codegen/src/parsers/typescript/index.js

```diff
@@ -13,19 +13,40 @@
 import type {SchemaType} from '../../CodegenSchema.js';
 
 const fs = require('fs');
-const {buildSchema} = require('./buildSchema');
+
+const {buildSchema} = require('../parsers-commons');
+const {Visitor} = require('./Visitor');
 const {TypeScriptParser} = require('./parser');
+const {buildComponentSchema} = require('./components');
+const {wrapComponentSchema} = require('./components/schema');
+const {buildModuleSchema} = require('./modules');
 
 const parser = new TypeScriptParser();
 
 function parseModuleFixture(filename: string): SchemaType {
   const contents = fs.readFileSync(filename, 'utf8');
 
-  return buildSchema(contents, 'path/NativeSampleTurboModule.ts', parser);
+  return buildSchema(
```

Comment:
same concern as above: I think we should only have 1 invocation of buildSchema, if possible, and `parseModuleFixture` should call `parseString`.

Reply:
Done ✅